ansible/roles/github-bot/templates/environment-file.j2

```diff
@@ -0,0 +1,14 @@
+NODE_ENV=production
+PORT={{ envs.port }}
+TRAVIS_TOKEN={{ envs.travis_token }}
+GITHUB_TOKEN={{ envs.github_token }}
+GITHUB_WEBHOOK_SECRET={{ envs.github_webhook_secret }}
+LOGIN_CREDENTIALS={{ envs.login_credentials }}
+NODE_REPO_DIR=/home/{{ server_user }}/repos/node
+LOGS_DIR=/home/{{ server_user }}/logs
+JENKINS_API_CREDENTIALS={{ envs.jenkins_api_credentials }}
+JENKINS_JOB_CITGM={{ envs.jenkins_job_citgm }}
+JENKINS_BUILD_TOKEN_CITGM={{ envs.jenkins_build_token_citgm }}
+JENKINS_JOB_NODE={{ envs.jenkins_job_node }}
+JENKINS_BUILD_TOKEN_NODE={{ envs.jenkins_build_token_node }}
+JENKINS_WORKER_IPS=147.75.70.237,147.75.73.189,169.60.150.88
```

Comment:
I've noticed quite a lot of warnings in the bot logs recently, about Jenkins updates being pushed from unknown IPs:

```
51.15.200.62
51.15.218.201
147.75.74.174
147.75.193.230
163.172.186.154
```

Are those IPs valid from Jenkins workers?

Reply:
Listed them out below, all seem like valid workers. Do you have some example requests you could send me over IRC/email just to validate them?

---

`51.15.200.62` is `test-scaleway-ubuntu1604-armv7l-1`
`51.15.218.201` is `test-scaleway-ubuntu1604-armv7l-2`
`147.75.74.174` is `test-packenet-ubuntu1604-arm64-2`
`147.75.193.230` is `test-packetnet-centos7-arm64-1`
`163.172.186.154` is `test-scaleway-ubuntu1604-armv7l-3`

Reply:
Sent you a `curl` example on IRC.